projects/typescript-angular/README.md

```diff
@@ -0,0 +1,35 @@
+# Integrate `bpmn-visualization` in an Angular 14 project
```

Comment:
To have consistency with the other projects, I would prefer to rename the folder of the project with `typescript-vanilla-with-angular`.

Reply:
I don't think we can. "vanilla" is intended for projects that do not use any framework. Here we are using the Angular framework.